tests/e2e/tests/commands/new.ts

```diff
@@ -1,60 +0,0 @@
-import * as path from 'path';
-import * as fs from 'fs';
-import {ng} from '../../utils/process';
-import {deleteDir} from '../../utils/fs';
-
-export default function() {
-  const parentTestprojectDir = process.cwd();
-  const ngNewProjectName = 'new-test-project';
-
-  return Promise.resolve()
-
-    // The test setup already creates a project, let's clean that before running `ng new`.
-    .then(() => disableTestSetupProject(parentTestprojectDir))
-
-    // Run `ng new` and tests...
-    .then(() => process.chdir(parentTestprojectDir))
-    .then(() => ng('new', ngNewProjectName))
-    .then(() => process.chdir(path.join(parentTestprojectDir, ngNewProjectName)))
-
-    // Try to run the unit tests.
-    .then(() => ng('test', '--single-run'))
-
-    // Revert disabling test-setup project, and prepare folder so test cleanup doesn't fail.
-    .then(() => prepareGitForTestCleanup(parentTestprojectDir, ngNewProjectName));
-}
-
-const angularCliPaths = ['.git', 'angular-cli.json', 'package.json'];
-const fileRenameSuffix = '.ng-backup';
-
-// Change the project inherited from Setup into normal
-//  non angular-cli project folder, so we can call `ng new`.
-function disableTestSetupProject (parentTestprojectDir: string) {
-  return Promise.resolve()
-    .then(() => process.chdir(parentTestprojectDir))
-
-    // Rename the files that define the folder as an angular-cli project
-    //  and that interfere with creating a child one.    
-    .then(() => angularCliPaths.map(
-      filePath => path.join(parentTestprojectDir, filePath)
-    ))
-    .then((filaPaths) => filaPaths.forEach(
-        filePath => fs.rename(filePath, filePath + fileRenameSuffix)
-    ));
-}
-
-function prepareGitForTestCleanup (parentTestprojectDir: string, projectName: string) {
-  return Promise.resolve()
-    // The post-test cleanup breaks if there's another git project inside it.
-    .then(() => deleteDir(path.join(parentTestprojectDir, projectName)))
```

Comment:
This was failing in Windows because we were still in the directory I believe. The new commit changes directory first to address this.